website/content/en/docs/building-operators/helm/migration.md

```diff
@@ -149,22 +160,33 @@ Note that the following environment variables are no longer used.
 - `OPERATOR_NAME` is deprecated. It is used to define the name for a leader election config map. Operator authors should begin using `--leader-election-id` instead.
 - `POD_NAME` was used to enable a particular pod to hold the leader election lock when the Helm operator used the leader for life mechanism. Helm operator now uses controller-runtime's leader with lease mechanism, and `POD_NAME` is no longer necessary.
 
-## Exporting metrics
+### Exporting metrics
 
 If you are using metrics and would like to keep them exported you will need to configure
 it in the `config/default/kustomization.yaml`. Please see the [metrics][metrics] doc to know how you can perform this setup.
 
 The default port used by the metric endpoint binds to was changed from `:8383` to `:8080`. To continue using port `8383`, specify `--metrics-addr=:8383` when you start the operator.
 
-## Checking the changes
+### Verify the migration
+
+The project can now be deployed on cluster by running the command:
+
+```sh
+make deploy IMG=example.com/memcached-operator:v0.0.1
```

Comment:
For the helm-operator, we are initially guiding users to scaffold nginx-operator (https://sdk.operatorframework.io/docs/building-operators/helm/migration/#creating-a-new-project) in the migration guide. Can we modify this to nginx to have uniformity?

Reply:
IHMO: We need update here for nginx-operator as @varshaprasad96 suggested
And then, at the end of the doc put a note to describe that the users are able to check a final result example in the testdata of the Memcached scenario which is the same example adapted across the docs.

Reply:
If you look through all the helm docs, you'll notice a mix of memcached and nginx. I'll change this here, but these docs need to align on GVK.

Reply:
The helm testdata really should align with docs. I don't see a reason not to use memcached in the helm example, over nginx. Thoughts?